index.js

```diff
@@ -312,6 +379,24 @@ class PluginXrpPaychan extends PluginBtp {
       await this._reloadIncomingChannelDetails()
     }
 
+    if (!this._peerAddress) {
```

Comment:
this _connect function overrides the custom _connect in BTP to send the xrp_address subprotocol after all normal BTP listeners are registered, right?

Reply:
yeah